### PR TITLE
updated the plugin package and updated app remoting prefabs and scripts

### PR DESCRIPTION
**Pull request by satyapoojasama**

### CODE REVIEW

File: SampleShared/Samples/AppRemoting/Scripts/AppRemotingSample.cs [hunk 1]
```diff
@@ -75,6 +85,17 @@ private void Awake()
             }
 
             ShowConnection2DUI();
+
+            // In a multi-scene scenario, this game object is usually destroyed on exiting the scene it is in. 
+            // This would unsubscribe the app remoting events on scene transcition, having no way for the app to know when connection state changes.
+            // Not destroying the gameobject would allow the app to listen to app remoting events across scene transcitions.
```
[the remark this answers is not in the document]
Comment (satyapoojasama) [reply SampleShared/Samples/AppRemoting/Scripts/AppRemotingSample.cs:91]: Sure, thanks for the suggestion, Keaton. Also, this code is going to be replaced in a bit with approach #1 which Oliver was suggesting in the chat yesterday. I pushed it by mistake yesterday.  I am not sure if we want to keep this approach around.

[the remark this answers is not in the document]
Comment (satyapoojasama) [reply SampleShared/Samples/AppRemoting/Scripts/AppRemotingSample.cs:91]: sure, thanks Keaton. Although I am still wondering if we should expose this approach or approach #1  which Oliver suggested yesterday in the chat with TryGet.. in OnEnable(). @yl-msft  thoughts?

[the remark this answers is not in the document]
Comment (satyapoojasama) [reply SampleShared/Samples/AppRemoting/Scripts/AppRemotingSample.cs:91]: After some testing, it looks like in mutli-scene scenario with approach #3 when using TryGet.. and relying on subscribing to events back in OnEnable() does not work with latest StartListeningToPlayer() without some awkward workarounds. The expectation when using this method is when disconnected, it still needs to keep listening until StopListening() is used. This is different than how old Listen() works. It stops listening on disconnect.

But, with the above approach, during a scene transition, it kind of loses the previous listen session as the coroutine dies, so it stops listening on disconnected. restarting listen if disconnected after a scene switch works but it is ugly.

I believe, unfortunately the latest StartListeningToPlayer() does not work without workarounds using approach #2 and #3. it works straightforward only using "DoNotDestroyOnLoad()" in approach #1.

File: RemotingSample/ProjectSettings/ProjectSettings.asset [hunk 2]
```diff
@@ -182,7 +182,7 @@ PlayerSettings:
   androidSupportedAspectRatio: 1
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
-    Standalone: com.DefaultCompany.RemotingSample
+    Standalone: com.Microsoft.RemotingSample
```
[the remark this answers is not in the document]
Comment (keveleigh) [reply RemotingSample/ProjectSettings/ProjectSettings.asset:185]: Can we change it to `MicrosoftSample` or something then instead? I find `DefaultCompany` to be a bit unpolished

[the remark this answers is not in the document]
Comment (keveleigh) [reply RemotingSample/ProjectSettings/ProjectSettings.asset:185]: Actually yeah, looks like BasicSample uses `MicrosoftSample`

https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/blob/9bbc6a91d05f956c9f0ad507396b2b11a8fc4a7f/BasicSample/ProjectSettings/ProjectSettings.asset#L15

[the remark this answers is not in the document]
Comment (KBonds) [reply RemotingSample/ProjectSettings/ProjectSettings.asset:185]: This is fixed in this PR https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/pull/131/ which will be merged into this branch

**Review by KBonds — APPROVED** [review]

**Review by yl-msft — APPROVED** [review]

**Pull request merged**